process/block/metablock.go

```diff
@@ -1575,9 +1574,14 @@ func (mp *metaProcessor) createPeerInfo() ([]block.PeerData, error) {
 
 // ApplyBodyToHeader creates a miniblock header list given a block body
 func (mp *metaProcessor) ApplyBodyToHeader(hdr data.HeaderHandler, bodyHandler data.BodyHandler) (data.BodyHandler, error) {
-	log.Debug("started creating block header",
-		"round", hdr.GetRound(),
-	)
+	tm := core.NewTimeMeasure()
+	tm.Start("ApplyBodyToHeader")
+	defer func() {
+		tm.Finish("ApplyBodyToHeader")
+
+		log.Debug("time measurements", tm.GetMeasurements()...)
```

Comment:
in peer processor you put `"measurements checkForMissedBlocks"` for log title. Why don't use use the same pattern here ? It would be easier when looking over the logs

Reply:
done